tests/requirements-313.txt

```diff
@@ -15,7 +15,7 @@ markupsafe>=2.1.0
 # Depends on grpcio
 #google-cloud-pubsub<=2.1.0
 #google-cloud-storage>=1.24.0
-lxml>=4.9.2
+# lxml>=4.9.2
```

Comment:
What's the reason for this comment here?

Reply:
After I commented out this module from requirements-313.txt file, it works fine, otherwise pip installation is failing in the container
![image](https://github.com/user-attachments/assets/32f9f68e-dc72-46f2-ac55-5e98c5b27c55)

Reply:
OK.

Reply:
I don't see this change justified just yet. The error message posted above has nothing to do with `lxml`. That is simply a consequence of setting `export AUTOWRAPT_BOOTSTRAP='instana'` before `instana` is even installed.

Reply:
@Ferenc-, I'm sorry, but I didn't get your point. This error is coming from the `pip` install before any test execution. How will the setting of `AUTOWRAPT_BOOTSTRAP` impact this phase?

Reply:
I've reverted the change,  it's working on CircleCI. Probably problem was occurring because of arm64 architecture.